src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -165,6 +165,10 @@ public void stop() {
         turningMotor.set(0);
     }
 
+    public double getVelocity() {
```

Comment:
REmove this.  There is already a getDriveVelocity

Reply:
Done